hack/verify-structured-logging.sh

```diff
@@ -48,15 +48,28 @@ while IFS='' read -r line; do
   migrated_packages+=("$KUBE_ROOT/$line")
 done < <(cat "${migrated_packages_file}")
 
-
+echo "Running structured logging static check on migrated packages"
 ret=0
 GOOS=linux    logcheck "${migrated_packages[@]}" || ret=$?
 GOOS=windows  logcheck "${migrated_packages[@]}" || ret=$?
 
 if [ $ret -eq 0 ]; then
-  echo "Structured logging static check is passed :)."
+  echo "Structured logging static check passed on migrated packages :)"
+else
+  echo "Please fix above failures. You can locally test via:"
+  echo "hack/verify-structured-logging.sh"
+fi
+
+echo -e "\nRunning structured logging static check on all packages"
+ret=0
```

Comment:
We should not reset the failure from above

Reply:
Even if I remove this, next line will overwrite it.  If I use a new var, I can't return multiple error codes.
How do we handle this situation?

Reply:
We don't care about exact value of ret, we just care that `ret!=0`.

Running
```
cmd || ret=$?
```
Can only set ret to value other than 0, as `||` ensures that this code is only run on error .

Reply:
Ack, will not reset. Will return the last exit code.